jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/utils/Bpmn2Loader.java

```diff
@@ -30,7 +31,17 @@ public Bpmn2Loader(Class testClass) {
 
     public Definitions loadProcessFromJson(String fileName,
                                            String zOrderEnabled,
-                                           String bpsimDisplay) throws Exception {
+                                           String bpsimDisplay,
+                                           List<String> workitemNames) throws Exception {
+
+        List<String> baseWorkItemNames = new ArrayList<String>(Arrays.asList("Email",
```

Comment:
Please store this as private static final and reuse on both places from this PR.